test/Control/Error/Util.purs

```diff
@@ -21,9 +19,7 @@ maybeId = MaybeT <<< Identity
 testMaybe :: forall a. (a -> Boolean) -> Maybe a -> Boolean
 testMaybe f a = fromMaybe false (f <$> a)
 
-suite :: forall eff. Aff ( timer :: TIMER
-                         , avar :: AVAR
-                         , testOutput :: TESTOUTPUT | eff) Unit
+suite :: forall a. Free (TestF a) Unit
```

Comment:
Just cosmetics, but maybe [`TestSuite a`]( https://github.com/bodil/purescript-test-unit/blob/v6.0.0/src/Test/Unit.purs#L69) is a bit more readable?